app/presenters/api/v2/commodities/commodity_presenter.rb

```diff
@@ -65,8 +65,14 @@ def zero_mfn_duty?
           third_country_measures.size.positive? && third_country_measures.all?(&:zero_mfn?)
         end
 
+        def basic_duty_rate
+          if third_country_measures.one?
+            third_country_measures.first&.formatted_duty_expression
+          end
+        end
```

Comment:
so if there are > 1 `third_country_measures` the basic_duty_rate is `nil?`

Reply:
That's the situation that I've copied, yeah